libcxx/test/std/time/time.zone/time.zone.db/time.zone.db.tzdb/tzdb.members.pass.cpp

```diff
@@ -37,7 +37,13 @@ int main(int, const char**) {
   tzdb.version = "version";
   assert(tzdb.version == "version");
 
-  // TODO TZDB add the other data members
+  std::vector<std::chrono::time_zone>& zones = tzdb.zones;
+  zones.clear(); // is it a non-const member?
```

Comment:
IMO clearing is not necessary here and below, since we already store in a non-const ref.